src/storybook/components/title/title.scss

```diff
@@ -1,8 +1,8 @@
-@import "../styles/typography.scss";
+@import "../../styles/typography.scss";
 @import "../../../styles/global-css-settings.scss";
-@import "../styles/content-spacing";
+@import "../../styles/content-spacing";
 
-.monday-storybook-title {
+@mixin title-appearance{
```

Comment:
because in the welcome page we display text with title appearance but the text itself is not a h3